source/app/pages/documents/view.js

```diff
@@ -322,14 +288,20 @@ function Document({ currentPageNumber, dispatch, id, document, pageTitle, search
             {track === 'redaction' &&
             document.redactions &&
             Object.keys(document.redactions).length ? (
-              <div className={css.downloadButtons}>
-                <Button inverted onClick={clearReds}>
-                  Clear Redaction
-                </Button>
-                <Button className={css.downloadRedacted} onClick={downloadRedacted}>
-                ⬇ Redacted Doc
-                </Button>
-              </div>
+                <div className={css.downloadButtons}>
+                  <Button inverted onClick={clearReds}>
+                    Clear Redaction
+                  </Button>
+                  <Menu>
+                    <MenuButton as={Button}>
+                      ⬇ Redacted Doc
```

Comment:
The conversion to PDF/PNG takes a few seconds especially when it has to handle multi page files. Would it make sense to provide a visual cue tip or disable the button for a few seconds so that user knows the request has been received?

Reply:
Agreed that we should have a loading state. I feel that it would be wasted effort to have it in this PR as we will be moving this button to a preview modal shortly. Can include the loading state there.